fabric_plugin/tasks.py

```diff
@@ -111,22 +110,29 @@ def _run_task(task, task_properties, fabric_env):
 
 
 @operation
-def run_commands(commands, fabric_env=None, **kwargs):
-    """runs the provider 'commands' in sequence
+def run_commands(commands, fabric_env=None, use_sudo=False, **kwargs):
+    """Runs the provider 'commands' in sequence
 
     :param commands: a list of commands to run
     :param fabric_env: fabric configuration
     """
     with fabric_api.settings(**_fabric_env(fabric_env, warn_only=True)):
         for command in commands:
             ctx.logger.info('running command: {0}'.format(command))
-            result = fabric_api.run(command)
+            if use_sudo:
```

Comment:
a matter of taste but this could be replaced with
`run = fabric_api.sudo if use_sudo else fabric_api.run`